llmfoundry/data/packing.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         pad_token_id: int,
         padding_side: Literal['left', 'right'],
         max_leftover_bins_to_keep: Optional[int] = None,
+        is_profiling: bool = False,
```

Comment:
can you add a doc string for this class and its args? ty!

Reply:
Done